Formula/mighttpd2.rb

```diff
@@ -5,8 +5,8 @@ class Mighttpd2 < Formula
 
   desc "HTTP server"
   homepage "http://www.mew.org/~kazu/proj/mighttpd/en/"
```

Comment:
`The URL http://www.mew.org/~kazu/proj/mighttpd/en/ could use HTTPS rather than HTTP`